reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -165,6 +165,8 @@ Accept wildcard characters: False
 Returns an object representing the item with which you are working.
 By default, this cmdlet does not generate any output.
 
+When you use the **PassThru** parameter, `Compare-Object` omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```

Comment:
This paragraph is supposed the current paragraph that describes the -PassThru parameter.

Reply:
replaced